src/templates/partials/awselb.hbs

```diff
@@ -1,86 +1,51 @@
-{
-    "AWSTemplateFormatVersion": "2010-09-09",
-    "Description": "Generated {{output.date}}, {{{output.link}}}",
-    "Parameters": {
-        "SSLCertificateId": {
-            "Description": "The ARN of the SSL certificate to use",
-            "Type": "String",
-            "AllowedPattern": "^arn:[^:]*:[^:]*:[^:]*:[^:]*:.*$",
-            "ConstraintDescription": "SSL Certificate ID must be a valid ARN. http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-arns"
-        }
-    },
-    "Resources": {
-        "ExampleELB": {
-            "Type": "AWS::ElasticLoadBalancing::LoadBalancer",
-            "Properties": {
-                "Listeners": [
-                    {
-                        "LoadBalancerPort": "443",
-                        "InstancePort": "80",
-                        "PolicyNames": [
-                            "Mozilla-{{form.config}}-2015-03"
-                        ],
-                        "SSLCertificateId": {
-                            "Ref": "SSLCertificateId"
-                        },
-                        "Protocol": "HTTPS"
-                    }
-                ],
-                "AvailabilityZones": {
-                    "Fn::GetAZs": ""
-                },
-                "Policies": [
-                    {
-                        "PolicyName": "Mozilla-{{form.config}}-2015-03",
-                        "PolicyType": "SSLNegotiationPolicyType",
-                        "Attributes": [
+AWSTemplateFormatVersion: 2010-09-09
+Description: Mozilla ELB configuration generated {{output.date}}, {{{output.link}}}
+Parameters:
+  SSLCertificateId:
+    Description: The ARN of the ACM SSL certificate to use
+    Type: String
+    AllowedPattern: ^arn:aws:acm:[^:]*:[^:]*:certificate/.*$
+    ConstraintDescription: >
+      SSL Certificate ID must be a valid ACM ARN.
+      https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#genref-arns
+Resources:
+  ExampleELB:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    Properties:
+      Listeners:
+        - LoadBalancerPort: '443'
+          InstancePort: '80'
+          PolicyNames:
+            - Mozilla-{{form.config}}-v5-0
+          SSLCertificateId: !Ref SSLCertificateId
+          Protocol: HTTPS
+      AvailabilityZones:
+        Fn::GetAZs: !Ref 'AWS::Region'
+      Policies:
+        - PolicyName: Mozilla-{{form.config}}-v5-0
+          PolicyType: SSLNegotiationPolicyType
+          Attributes:
 {{#if (includes "SSLv3" output.protocols)}}
-                            {
-                                "Name": "Protocol-SSLv3",
-                                "Value": true
-                            },
+            - Name: Protocol-SSLv3
```

Comment:
Looking back on this, it looks like it might be better to do an each loop over output.protocols, since their format matches what AWS uses.

Reply:
Can I open an issue on that and solve it in a distinct PR?

Reply:
```
{{#each output.protocols}}
                            {
                                "Name": "{{this}}",
                                "Value": true
                            }
{{/each}}
```

Reply:
I've made this change